plugins/database/mysql/connection_producer.go

```diff
@@ -35,7 +35,7 @@ type mySQLConnectionProducer struct {
 	TLSCAData             []byte `json:"tls_ca"              mapstructure:"tls_ca"              structs:"-"`
 
 	// tlsConfigName is a globally unique name that references the TLS config for this instance in the mysql driver
-	tlsConfigName					string
+	tlsConfigName string
```

Comment:
Has `gofmt` been run on this file?  I'm not sure of the behavior when there is whitespace separating declarations.

Reply:
It has. This should reflect those runs. I'll double check though.

Reply:
Double checked, only thing it found was an extra line of whitespace at the end of a file. Just reminds me that I need to run it before I commit.

Reply:
`gofmt` doesn't line up the types if there is a newline or a comment between them. Basically it groups fields together and if anything interrupts the group it resets to a new group.